starcheck/src/starcheck

```diff
@@ -8,12 +8,15 @@
 # Unset PYTHONPATH because custom PYTHONPATH should never be allowed
 # for flight starcheck
 unset PYTHONPATH
-export PYTHONHOME=$SKA
+
+PYTHON_EXE=`which python`
+PYTHON_DIR=`dirname $PYTHON_EXE`
+export PYTHONHOME=`dirname $PYTHON_DIR`
 
 # Check for skare3 perl
 if [[ ! -f ${PYTHONHOME}/bin/perl ]];
 then
-    echo "skare3 perl not installed. conda install perl perl-core-deps perl-ska-classic"
+    echo "skare3 perl not installed. conda install perl perl-ska-convert"
```

Comment:
It probably makes more sense to just make a metapackage that has the perl-starcheck-needs, but this change should be fine for now even if it isn't consistent with the similar printent statement a few lines below.